backend/src/main/java/gov/cdc/usds/simplereport/idp/authentication/LiveOktaAuthentication.java

```diff
@@ -137,21 +142,22 @@ public UserAccountStatus getUserStatus(String activationToken, String userId, St
    * https://developer.okta.com/docs/reference/api/authn/#request-example-for-activation-token
    * https://developer.okta.com/docs/reference/api/authn/#response-example-for-activation-token-success-user-without-password
    */
+  @Override
   public String activateUser(String activationToken, String crossForwardedHeader, String userAgent)
       throws InvalidActivationLinkException {
     JSONObject requestBody = new JSONObject();
     requestBody.put("token", activationToken);
-    String authorizationToken = "SSWS " + _apiToken;
+    String authorizationToken = "SSWS " + apiToken;
     HttpHeaders headers = new HttpHeaders();
     headers.setContentType(MediaType.APPLICATION_JSON);
     headers.setAccept(List.of(MediaType.APPLICATION_JSON));
     headers.add(HttpHeaders.USER_AGENT, userAgent);
     headers.add("X-Forwarded-For", crossForwardedHeader);
     headers.add("Authorization", authorizationToken);
     HttpEntity<String> entity = new HttpEntity<>(requestBody.toString(), headers);
-    String postUrl = _orgUrl + "/api/v1/authn";
+    String postUrl = orgUrl + "/api/v1/authn";
     try {
-      String response = _restTemplate.postForObject(postUrl, entity, String.class);
+      String response = restTemplate.postForObject(postUrl, entity, String.class);
```

Comment:
is it possible to also change this to the sdk, rather than manual api calls?

Reply:
This would be available in the [AuthN sdk](https://github.com/okta/okta-auth-java), but it seems like they will not update that SDK to support spring boot 3 nor will it support okta-sdk-api versions greater than 8. 
I believe their suggestion is to move to the new [Okta Idenetity engine](https://developer.okta.com/docs/concepts/oie-intro/), but it seems like that would be another significant lift and would need more research before moving in that direction.